Source/Modules/python.cxx

```diff
@@ -819,7 +819,8 @@ class PYTHON:public Language {
     Printf(f_wrappers, "%s\n", methods);
     Append(methods_proxydocs, "\t { NULL, NULL, 0, NULL }\n");
     Append(methods_proxydocs, "};\n");
-    Printf(f_wrappers, "%s\n", methods_proxydocs);
+    if (fastproxy)
+      Printf(f_wrappers, "%s\n", methods_proxydocs);
```

Comment:
Looks good, except the only use of this seems to be in `Lib/python/pyinit.swg` inside a conditional:

```
#if defined SWIGPYTHON_FASTPROXY && !defined SWIGPYTHON_BUILTIN
```

So I wonder if the test here should really be `(fastproxy && !builtin)` instead?  `builtin` can be true or false here, as shown by the test just below.

Reply:
Oh, I see you tried that in a previous iteration and it failed:

https://github.com/swig/swig/runs/5394070693?check_suite_focus=true

Reply:
Looks like `((fastproxy && !builtin) || have_fast_proxy_static_member_method_callback)` is the exact condition it's used under, or did you already check that and find it doesn't work?  It's hard to fully see what's been tried with all the force-pushing.

Reply:
you're right